Main.hs

```diff
@@ -105,10 +105,7 @@ main = do
     dieWithUsage
 
   when verbose $
-    printf "GHC iserv starting (in: %d; out: %d)\n"
```

Comment:
Old: GHC iserv starting (in: 79; out: 78)
New: GHC iserv starting (in: {handle: <file descriptor: 79>}; out: {handle: <file descriptor: 78>})

Is preserving the old behavior worth CPP?

Reply:
@alexfmpe you mean the output only? I don't think it is. Iserv-proxy and interpreter are primarily for functionality. I hope that with the cross support in ghc we can setup proper testing here.